.github/workflows/test.yaml

```diff
@@ -24,22 +28,44 @@ defaults:
   run:
     shell: bash
 
+env:
+  # Set to 'yes' to open a tunnel to GitHub's VMs through tmate on failures.
+  # Also increase timeout-minutes for the relevant OS when debugging remotely
+  # and you need a longer session.
+  TMATE_DEBUG: 'no'
```

Comment:
We only want to enable tmate when we are to start a debug session...and not on any failure.

tmate is only triggered on failure